src/libs/signer.ts

```diff
@@ -0,0 +1,36 @@
+import { ethers } from 'ethers';
+import Web3 from 'web3';
+
+export function getAddress(): Promise<string> {
+  // await window.ethereum.enable();
+  const provider = new ethers.providers.Web3Provider(Web3.givenProvider);
+  const signer = provider.getSigner();
+  return signer.getAddress();
+}
```

Comment:
Should this function create a provider each time? This feels like it should be fetched from the wallet context

Reply:
Yeah, fair enough. I think this may just be a web3 boilerplate helper function. But I think I can just remove it since, `onboardjs` provides an address in the wallet context.

Reply:
Fixed in a59ac9b57bade73a60925b772d6b4323f3221b3d